lib/CapabilityDelegation.js

```diff
@@ -42,11 +42,13 @@ module.exports = class CapabilityDelegation extends ControllerProofPurpose {
    *   `utils.verifyCapabilityChain`.
    * @param [currentDate = new Date()] {Date} - The date used for comparison
    *   when determining if a capability has expired.
+   * @param [maxChainLength=10] - The maximum length of the capability
+   *   delegation chain.
```

Comment:
- `currentDate` and `maxChainLength` have listed defaults here, but not in the code.
- `maxTimestampDelta` should maybe be using `[maxTimestampDelta=Infinity]` rather than listing default in the text?

Reply:
The defaults are set in the common helper.

Reply:
Ok. I think doc strings should just document a function vs what happens if you call it from elsewhere.